Automattic/newspack-plugin
# feat(ia): general restructuring

### All Submissions:


### Changes proposed in this Pull Request:

Are intended to provide a solid foundation to rollout the IA feature. Some pain points that lead to these changes:
* Limited configurability with regards to module bundling and preprocessing. Most routes resulted in severe duplication (specifically around `commons` `splitChunks`)
* Separation of each admin page into individual React apps, was/is unnecessary. Changes consolidate these into a single entry-point which can take advantage of caching and single source of truth makes it easier to maintain.

### How to test the changes in this Pull Request:

1. Checkout and build these changes `git checkout feat/ia-general && npm run build`.
2. Navigate to Newspack / Dashboard & Newspack / Settings and confirm both resources load correctly.

### Other information:
